emulators/xhyve/files/add-hypervisor-elements.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/xhyve-entitlements.plist b/src/xhyve-entitlements.plist
```

Comment:
This has been done upstream.  Can you drop the patch and pick up newer upstream?